templates/prometheus.yaml

```diff
@@ -12,6 +12,14 @@ spec:
       ---
       prometheus:
         prometheusSpec:
+          externalUrl: "/ops/portal/prometheus"
+          storageSpec:
+            volumeClaimTemplate:
+              spec:
+                accessModes: ["ReadWriteOnce"]
+                resources:
+                  requests:
+                    storage: 50Gi
```

Comment:
Is 50Gi a recommendation from Prometheus or a default of their's somewhere? When we make decisions like this if we can add a comment and a reference which explains the why it can be helpful for future maintainers, could we add something helpful here to explain the storage needs here?